docs/destinations/forwarding-events/snowbridge/concepts/scaling/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Scaling"
+date: "2023-02-27"
+sidebar_position: 150
+---
+
+# Scaling
+
+## Horizontal scaling
+
+Snowbridge is built to suit a horizontal scaling model, and you can safely deploy multiple instances of Snowbridge to consume the same input out-of-the-box (the kinesis source requires Dynamo DB tables to be created for this purpose). No addditional configuration or setup is required for the app to smoothly run across multiple instance/environment, compared to a single instance/environment.
```

Comment:
```suggestion
Snowbridge is built to suit a horizontal scaling model, and you can safely deploy multiple instances of Snowbridge to consume the same input out-of-the-box. No addditional configuration or setup is required for the app to smoothly run across multiple instance/environment, compared to a single instance/environment.

:::note

If you are using the Kinesis source, Snowbridge will automatically create Dynamo DB tables to manage multiple instances reading from the same stream.

:::
```

Reply:
It doesn't do that - but I've edited with a link to setup instructions.